Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -1,645 +1,647 @@
-                                     .HasColumnType("int")
-                                     .HasAnnotation("SqlServer:ValueGenerationStrategy",
-                                                    SqlServerValueGenerationStrategy.IdentityColumn);
+            modelBuilder.Entity("Models.RoleScope", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("int")
+                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
-                                    b.Property<int>("RoleId")
-                                     .HasColumnType("int");
+                    b.Property<int>("RoleId")
+                        .HasColumnType("int");
 
-                                    b.Property<string>("Scope")
-                                     .HasColumnType("nvarchar(max)");
+                    b.Property<string>("Scope")
+                        .HasColumnType("nvarchar(max)");
 
-                                    b.HasKey("Id");
+                    b.HasKey("Id");
 
-                                    b.HasIndex("RoleId");
+                    b.HasIndex("RoleId");
 
-                                    b.ToTable("RoleScope");
-                                });
+                    b.ToTable("RoleScope");
+                });
 
-            modelBuilder.Entity("Models.User",
-                                b =>
-                                {
-                                    b.Property<int>("Id")
-                                     .ValueGeneratedOnAdd()
-                                     .HasColumnType("int")
-                                     .HasAnnotation("SqlServer:ValueGenerationStrategy",
-                                                    SqlServerValueGenerationStrategy.IdentityColumn);
+            modelBuilder.Entity("Models.Tag", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("int")
+                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
-                                    b.Property<string>("Email")
-                                     .IsRequired()
-                                     .HasColumnType("nvarchar(max)");
+                    b.Property<string>("Name")
+                        .HasColumnType("nvarchar(max)");
 
-                                    b.Property<string>("IdentityId")
-                                     .IsRequired()
-                                     .HasColumnType("nvarchar(max)");
+                    b.HasKey("Id");
 
-                                    b.Property<int?>("InstitutionId")
-                                     .HasColumnType("int");
+                    b.ToTable("Tag");
+                });
```

Comment:
Not sure why this is here, can you check if this will cause any problems?

Reply:
Dex seems to work fine after deletion of this file.. Should I push it? @niraymak

Reply:
https://www.learnentityframeworkcore.com/migrations/model-snapshot I Dont think we should though